tdr-snapshot-examples/get_snapshot_with_drs_and_download.py

```diff
@@ -0,0 +1,54 @@
+# This file goes through the full flow of getting a snapshot,
+# getting it's report's drs info, and then downloading the report
+# to the file system.
+
+import json
+
+from get_file_info_by_drs_path import get_file_info_by_drs_path
+from get_snapshot import get_snapshot
+from obtain_token import obtain_session
+
+
+tdr_domain = "data.terra.bio"
+
+headers = {
+    "User-Agent": "GPO Bound Query",
+    "Content-Type": "application/json",
+}
+
+
+def download_file_from_drs(session, drs_json):
+    resp = session.get(url=drs_json["accessUrl"]["url"], headers=headers)
+    with open(f"{drs_json['fileName']}", "wb") as report_file:
+        report_file.write(resp.content)
+        report_file.close()
+
+
+if __name__ == "__main__":
+    session = obtain_session()
+    print("==>Snapshot Information")
+    snapshot_data = get_snapshot(session)
```

Comment:
This is especially an example of why passing the snapshot ID in the method would be best.